design/baremetal-operator/configmap_for_host_inspection_data.md

```diff
@@ -0,0 +1,193 @@
+# hostData custom resource for host inspection data
+
+## Status
+
+implementable
+
+## Summary
+
+Introduce HostData Custom Resource (CR) to store host inspection data.
+
+## Motivation
+
+Currently inspection data is written to baremetalhost status subresource. When
+pivoting to a target cluster, with *`clusterctl move`*, status subresource is
+not copied with the host, because from Cluster-API perspective, it's controllers'
+job to rebuild the status in the target cluster. Also, according to the [Kubernetes API convention](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status), status is expected to be reconstructed by controllers if
+it's lost for any reason. Baremetal Operator (BMO) is capable of reconstructing
+host status by re-inspecting the host, but it will force the already provisioned
+host to go through another round of provisioning, which eventually reboots the
```

Comment:
I would say another round of inspection

Reply:
I think it is also important to let people know that the host will be rebooted, which is I think when host is provisioning?

Reply:
yes true